INSTALLATION.md

```diff
@@ -0,0 +1,161 @@
+```
+
+- Instale o geckodriver
+
+```
+wget https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-linux64.tar.gz
+tar -xvzf geckodriver*
+sudo chmod +x geckodriver
+sudo mv geckodriver /usr/local/bin/
+rm geckodriver-v0.26.0-linux64.tar.gz
+```
+
+- Instale o ruby utilizando um gerenciador de versões a sua escolha (rvm ou rbenv, por exemplo)
+
+    - Utilizando rvm
+
+    ```
+    sudo apt-get install libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
+    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+    curl -sSL https://get.rvm.io | bash -s stable
+    source ~/.rvm/scripts/rvm
+    rvm install 2.7.0
+    rvm use 2.7.0 --default
+    ruby -v
+    ```
+
+    - Utilizando rbenv
+
+    ```
+    git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+    echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+    echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+    exec $SHELL
+
+    git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+    echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+    exec $SHELL
+
+    rbenv install 2.7.0
+    rbenv global 2.7.0
+    ruby -v
+    ```
+
+- Instale o bundler e as gems do projeto
+
+```
+gem install bundler
+bundle install
+```
+
+> Mac OS X:
+
+- Instale o geckodriver
+
+```
+brew cask install firefox
+wget https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-macos.tar.gz
+tar -xvzf geckodriver*
+sudo chmod +x geckodriver
+sudo mv geckodriver /usr/local/bin/
+rm geckodriver-v0.26.0-macos.tar.gz
+```
+
+- Instale o ruby utilizando um gerenciador de versões a sua escolha (rvm ou rbenv, por exemplo)
+
+    - Utilizando rvm
+
+    ```
+    sudo gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+    curl -sSL https://get.rvm.io | sudo bash -s stable
+    sudo usermod -a -G rvm `whoami`
+    if sudo grep -q secure_path /etc/sudoers; then sudo sh -c "echo export rvmsudo_secure_path=1 >> /etc/profile.d/rvm_secure_path.sh" && echo Environment variable installed; fi
+
+
+    rvm install 2.7.0
+    rvm use 2.7.0 --default
+    ruby -v
+    ```
+
+    - Utilizando rbenv
+
+    ```
+    brew install rbenv ruby-build
+
+    # Add rbenv to bash so that it loads every time you open a terminal
+    echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+    source ~/.bash_profile
+
+    # Install Ruby
+    rbenv install 2.7.0
+    rbenv global 2.7.0
+    ruby -v
+    ```
+
+- Instale o bundler e as gems do projeto
+
+    ```
+    gem install bundler
+    bundle install
+    ```
+
+### Executando os testes do corretor
+
+- Antes de executar os testes do corretor é necessário clonar o repositório a ser corrigido dentro do diretório `spec`
+
+    ```
+    cd spec/
+    git clone git@github.com:tryber/<project-repo-name>.git
```

Comment:
If we have just cloned the repo inside the spec folder, why do we need to pass the repo name as an env var on the next step?

Reply:
In this case, it's because Capybara needs to know the path to be visited when running the specs.

For instance, see the test below:
```ruby
RSpec.describe do
  let!(:index_path) { "/#{ENV['GITHUB_REPOSITORY']}/index.html" }

  describe 'project requirements' do
    it 'O corpo da página deve possuir uma cor (diferente da cor branca) como cor de fundo' do
      visit index_path

      background_color = page.find('body').native.css_value('background-color')

      expect(background_color).not_to be_empty
      expect(background_color).not_to eq('rgb(255, 255, 255)')
    end

    # ...

  end
end
```

Reply:
It will always be the *one* folder inside `spec`, wouldn't it ? :thinking:

Reply:
When running inside the GitHub Action, yes, and this ENV VAR guides the paths to move the files around in the Docker image. When running locally, probably, but not necessarily.

What do you suggest?

Reply:
I'm feeling that this is an unnecessary redundancy. 

But if you think we must have it, let's move on.